packages/contracts-rfq/CHANGELOG.md

```
@@ -3,6 +3,93 @@
+### Bug Fixes
+
+* **contracts-rfq:** limit the amount of solhint warnings [SLT-245] ([#3182](https://github.com/synapsecns/sanguine/issues/3182)) ([c15ec86](https://github.com/synapsecns/sanguine/commit/c15ec8691902817f096589553bac360b53ba40cf))
+
+
+
+
+
+# 0.5.0 (2024-09-24)
+
+
+### Bug Fixes
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)